common/source/docs/common-efi.rst

```diff
@@ -22,6 +22,7 @@ The following units have been tested and are supported:
 - Lutan Serial
 - DroneCAN (adapted versions of the above serial units)
 - PiccoloCAN (`Currawong's ECU <https://www.currawong.aero/ecu/>`__ and `IntelliJect EFI <https://power4flight.com/uav-engine-products/uav-engine-control-units/intelliject-efi/>`__)
+- Hirth Engines
```

Comment:
can add the link of the provider https://hirthengines.com/2-stroke-engines/41-series/

Reply:
then I will need to add links for the others also...which I will do